src/main/java/commands/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package commands;
+
+public class Deadline extends Task {
+    protected String date;
+
+    public Deadline(String name) {
+        super(name);
+        this.date = "None";
+    }
+
+    public Deadline(String name, String date) {
+        super(name);
+        this.date = date;
+    }
+
+    public static String parseName(String command) {
+        return command.substring(command.indexOf(" ") + 1, command.indexOf("/")-1);
```

Comment:
There are spaces between the command.indexOf(" ") + 1, but not spaces between command.indexOf("/"-1), try to maintain same coding standards throughout!